bundles/http/Citrix/5c20233e-ead7-4385-a3f9-c703f194514b/5b0d4f89-2cfb-499b-8f63-f70cbed8fa13.js

```diff
@@ -0,0 +1,136 @@
+const ENDPOINT = "/incidents";
+const LIMIT = 100;
+let moment = library.load("moment-timezone");
+
+integration.define({
+    "synchronizations": [
+        {
+            "name": "UpdateComments",
+            "fullSyncFunction": updateComments,
+            "incrementalSyncFunction": updateComments
+        }
+    ],
+    "model": {
+        "tables": [
+            {
+                "name": "updated_comments",
+                "columns": [
+                    {
+                        "name": "id",
+                        "type": "INTEGER",
+                        "primaryKey": true
+                    },
+                    {
+                        "name": "body",
+                        "type": "STRING",
+                        "length": 100
+                    },
+                    {
+                        "name": "created_at",
+                        "type": "DATETIME"
+                    },
+                    {
+                        "name": "commenter_id",
+                        "type": "INTEGER"
+                    },
+                    {
+                        "name": "user_name",
+                        "type": "STRING"
+
+                    },
+                    {
+                        "name": "user_email",
+                        "type": "STRING"
+                    }
+                ]
+            }
+        ]
+    }
+});
+
+function replaceBodyCharacters(key, value) {
+    let str = value;
+    if (key === "body") {
+        str = str.replace(/<[^>]+>/gm, '');
+        str = str.replace('\n&nbsp;', '');
+    }
+    return str;
+}
+
+async function updateComments({dataStore, client, latestSynchronizationTime}) {
+    let incidentComments = "";

```

Comment:
Why it is not array?

Reply:
It is just another way to do.

Reply:
So if there is no special reason then, please rewrite it to array.
Thank you.

Reply:
Changed to array.